Composer/packages/form-dialogs/src/components/VisualEditor.tsx

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Consider calling this something other than VisualEditor as that is the name of the component for the trigger/action flow editing.  FormDialogEditor? FormBuilder?

Reply:
FormDialogPropertiesEditor?